packages/core/src/Stores/client-store.js

```diff
@@ -1028,6 +1027,7 @@ export default class ClientStore extends BaseStore {
         } else {
             await WS.forgetAll('balance');
             await BinarySocket.authorize(this.getToken());
+            if (this.root_store.common.has_error) this.root_store.common.setError(false, null);
```

Comment:
will this overrides authorize error? can we have a specific check to see if current platform is trader and its maltainvest ?

Reply:
This won't override other errors as it's happening at the very first of our stores' life-cycle. It's just after this that `onMount` for related stores is called.